phoenicis-javafx/src/main/java/org/phoenicis/javafx/controller/MainController.java

```diff
@@ -58,6 +58,7 @@ public MainController(String applicationName, LibraryController libraryControlle
         appsController.setOnAppLoaded(() -> {
             enginesController.loadEngines();
             containersController.loadContainers();
+            libraryController.updateLibrary();
```

Comment:
Is this method called at another location too?
I'm just asking because you didn't remove another call to `updateLibrary`